emulator/src/components/hdw-tft/hdw-tft.c

```diff
@@ -280,7 +280,7 @@ void drawDisplayTft(fnBackgroundDrawCallback_t fnBackgroundDrawCallback)
                     int colorolor = frameBuffer[(y * TFT_WIDTH) + x];
                     if (colorolor == 216)
                     {
-                        colorolor = c500;
```

Comment:
This was intentional to highlight when transparent pixels are drawn, because they shouldn't be. Maybe I'm not understanding the use case, but shouldn't wsgs with transparency be drawn with transparent functions, and vice versa?

Reply:
And this is specific to the emulator, so this is supposed to be a visual warning. Does the fact that you added an explicit value for `cTransparent` to `paletteColors[]` make this wholly unnecessary?

Reply:
The idea is to gain some FPS by using `drawWsgTile` to draw the tilemap instead of `drawWsgSimple`. 

All the tiles were designed with transparent pixels, which when normally drawn with `drawWsgTile`, appear as an off-white color. But, since the background in Pango is always black, we can make it look correct by adding an explicit definition for `cTransparent`, setting it to black.

The emulator handles `cTransparent` separately in this `if` statement here, but I should probably just add an explicit definition to the emulator palette instead...

Reply:
Of course, if we can further optimize the MIDI playback, I can go without the `cTransparent` changes.

Reply:
As long as it's consistent and expected, the change is fine. I'd add to the emulator palette and call it a day.

Reply:
I'll add it to the emulator palette rn actually